lib/src/main/java/vmath/Mat2.java

```diff
@@ -10,6 +10,27 @@ public Mat2() {
         };
     }
 
+    public Mat2(float[] initialValues) {
+        if (initialValues.length != 4) {
+            throw new IllegalArgumentException("Initial values array must have exactly 4 elements.");
+        }
```

Comment:
Replace the message with "Array length must be 4 for a 2x2 matrix." This change must be made to maintain overall code consistency.